### PR TITLE
Added rock relevance filter

### LINKED ISSUE #268: Add filter Relevance to rock filter dialog
Subtask of #251

### PR DESCRIPTION
**Pull request by paetz**
Resolves #268

### CODE REVIEW

File: app/src/main/java/com/yacgroup/yacguide/database/comment/RockComment.kt [hunk 1]
```diff
@@ -32,9 +32,12 @@ class RockComment {
 
     companion object {
 
+        const val RELEVANCE_NONE = 0
+
         // This needs to be in sync with sandsteinklettern.de!
-        val QUALITY_MAP: Map<Int, String> = object : HashMap<Int, String>() {
+        val RELEVANCE_MAP: Map<Int, String> = object : LinkedHashMap<Int, String>() {
```
Comment (paetz) [inline app/src/main/java/com/yacgroup/yacguide/database/comment/RockComment.kt:38]: Note: `LinkedHashMap` preserves input order

File: app/src/main/res/layout/search_dialog.xml [hunk 2]
```diff
@@ -33,6 +32,31 @@
         android:maxLines="1"
         android:singleLine="true"/>
 
+    <View style="@style/ThinDivider"/>
```
Comment (paetz) [inline app/src/main/res/layout/search_dialog.xml:35]: I can see those dividers in the IDE but not in the emulator. Have you got an idea why?

Comment (chrgernoe) [reply app/src/main/res/layout/search_dialog.xml:35]: However, in general the dialog does not look very nice with these dividers. Please add at least some left and right space as in the `RecyclerView`.

**Review by chrgernoe — CHANGES_REQUESTED** [review]
Please consider the comment I made regarding the dialog style.

**Comment by paetz** [comment]
> Please consider the comment I made regarding the dialog style.

Thx for the solution with the separators. I added left and right margin accordingly.

File: app/src/main/res/values/styles.xml [hunk 3]
```diff
@@ -51,6 +49,9 @@
 
     <style name="AppTheme.Dialog" parent="ThemeOverlay.AppCompat.Dialog">
         <item name="android:listDivider">@color/colorSecondary</item>
+        <item name="android:layout_marginLeft">8dp</item>
+        <item name="android:layout_marginRight">8dp</item>
+
```
Comment (chrgernoe) [inline app/src/main/res/values/styles.xml:54]: Please remove this empty line before merging.

**Review by chrgernoe — APPROVED** [review]

**Pull request merged**